packages/zoe/test/unitTests/setupNonFungibleMints.js

```diff
@@ -0,0 +1,39 @@
+import harden from '@agoric/harden';
+
+import produceIssuer from '@agoric/ertp';
+
+const setupNonFungible = () => {
+  const ccBundle = produceIssuer('CryptoCats', 'strSet');
+  const rpgBundle = produceIssuer('MMORPG Items', 'strSet');
```

Comment:
Could one of these use a `set` mathHelper?

Reply:
done